frontend/src/main/components/Leaderboard/LeaderboardTable.js

```diff
@@ -16,6 +16,13 @@ export default function LeaderboardTable({ leaderboardUsers , currentUser }) {
         {
             Header: 'Total Wealth',
             accessor: 'totalWealth',
+            //this is the func to  can return custom tableCell
+            Cell : (props)=>{
+            //props.value will contain your date
+            //you can convert total wealth data here
+                const custom_wealth = '$'+props.value.toFixed(2)
+                return <span>{custom_wealth}</span>
```

Comment:
If you do this, it should display right aligned, which makes more sense for currency.  

```suggestion
                return div style={{textAlign: "right"}}>{custom_wealth}</div>
```